.github/labeler.yml

```
@@ -0,0 +1,55 @@
+#
+# Add project labels
+#
+
+# Add 'Documentation' label to any changes in the documentation
+
+'Documentation':
+  - documentation/**
+  - CONTRIBUTING.md
+  - DEBUGING.md
+  - DEV-INTRO.md
+  - README.md
+  - book_source/**
+
+# Add 'Dockerfile' label to any changes in the docker directory
+'Dockerfile':
+  - docker/**
+
+
+# Add 'Website' label to any changes in the web directory
+
+'Website':
+  - web/**
+
+# Add 'Base' label to any changes in the base directory
+
+'Base':
+  - base/**
+
+# Add 'Models' label to any changes in the models directory
+
+'Models':
+  - models/**
+
+# Add 'Modules' label to any changes in the modules directory
+
+'Modules':
+  - modules/**
+
+# Add 'GitHub Actions' label to any changes in the .github/workflows directory
+
+'GitHub Actions':
+  - .github/workflows/**
+
+# Add 'Scripts' label to any changes in the scripts directory
+
+'Scripts':
+  - scripts/**
+
+# Add 'Tests' label to any changes in the tests directory
+
+'Tests':
+  - tests/**
```

While we're at it, might as well tag in-package unit tests too (unlike my point above about CHANGELOG, I think it probably *is* a good thing for PRs that touch a test directory to be labeled 'tests' even if that's not their main point)
```suggestion
  - tests/**
  - '**/tests/**'
  - '!**/tests/Rcheck_reference.log'
```